### ISSUE TITLE
Autofocus to modals should be added

### ISSUE THREAD

**Issue by ElenaGrebenchuk** [epam/UUI#15]
# Expected Behavior

Autofocus should be automatically added to modals of any type, because of accessibility and to avoid any communications between user and locked by modal screen,  prevent changing the data in text fields where he was focused before. 

# Current Behavior

There is now autofocus on modal. User is able to proceed changing data in text fields.

**Comment by AlekseyManetov** [comment]
You need to use `autoFocus={ true }` prop for inputs inside modal, to get such behavior.

For example:

`
<ModalWindow>
    <TextInput value={ value } onValueChange={ handleChange } autoFocus={ true }>
</ModalWindow>
`

Modal can't get focus on itself, because it's not focusable element.
